operator/integration-tests/model-in-image/run-test.sh

```diff
@@ -54,6 +54,7 @@ function usage() {
     WORKDIR               : /tmp/\$USER/mii-sample-work-dir
     DOMAIN_NAMESPACE      : sample-domain1-ns
     MODEL_IMAGE_NAME      : model-in-image 
+    MODEL_IMAGE_TAG       : ${WDT_DOMAIN_TYPE}-v2
```

Comment:
Defaults to JRF-v1, JRF-v2, WLS-v1, or WLS-v2 depending on use-case and domain type.

Reply:
(` ${WDT_DOMAIN_TYPE}-v2` is incorrect because WDT_DOMAIN_TYPE is hard coded to WLS when , and v2/v1 varies based on the situation).

Reply:
```
      MODEL_IMAGE_TAG       : Defaults to JRF-v1, JRF-v2, WLS-v1, or WLS-v2 
                                                  depending on use-case and domain type.
                                                  IMPORTANT: If setting this env var, do not
                                                  run multiple use cases on the same command line,
                                                  as then all use cases will use the same image.
```

Reply:
fixed